querqy-for-lucene/querqy-solr/src/main/java/querqy/solr/QuerqyRewriterRequestHandler.java

```diff
@@ -140,7 +140,7 @@ public SolrParams params() {
 
     public static final String DEFAULT_HANDLER_NAME = "/querqy/rewriter";
 
-    private RewriterContainer<?> rewriterContainer = null;
+    protected RewriterContainer<?> rewriterContainer = null;
```

Comment:
I'd prefer not to make this visible as it might change state (before/after init()).

Also, the RewriterContainer has a close() method and the QuerqyRewriterRequestHandler is responsible for calling it, which we cannot guarantee if someone else changed the value of this variable, so let's not expose it. Maybe a getter method would be okay. (Unfortunately we forgot to register the RewriterContainer as a close hook with the SolrCore in QuerqyRewriterRequestHandler.init() -> I'll file a bug for it. See ZkRewriterContainer for an example what needs to be done on close())

Reply:
See https://github.com/querqy/querqy/issues/177 for the bug I've mentioned.

Reply:
Re testing: I think you can just use a ZK client (like in querqy.solr.ZkRewriterContainerFileSizeTest) to check where the config is stored